ipatests/test_webui/ui_driver.py

```diff
@@ -732,8 +732,15 @@ def button_click(self, name, parent=None,
     def _button_click(self, selector, parent, name=''):
         btn = self.find(selector, By.CSS_SELECTOR, parent, strict=True)
         self.move_to_element_in_page(btn)
+
+        # The small timeout (up to 1 second) allows to prevent exceptions when
+        # driver attempts to click a button before it is rendered.
+        WebDriverWait(self.driver, 1, 0.2).until(
```

Comment:
I suggest to increase max wait time to 5 seconds - we execute tests in VMs on heavily loaded hosts and I feel that browser can get stuck for second or two.

Reply:
Yes, agree, fixed.